.github/workflows/lint.yml

```diff
@@ -49,10 +39,10 @@ jobs:
               STATUS=1
             fi
           }
-
-          assert-nothing-changed go fmt ./...
           assert-nothing-changed go mod tidy
-
-          bin/golangci-lint run --out-format=colored-line-number --timeout=3m || STATUS=$?
```

Comment:
[`colored-line-number` has been enabled by default](https://golangci-lint.run/product/migration-guide/#outputformatsformat-colored-line-number).

`--out-format` seems to have been removed.